hs/src/Reach/Verify/SMT.hs

```diff
@@ -1344,6 +1349,20 @@ smt_n = \case
         smt_invblock (B_Assume True) inv
         smt_invblock (B_Assume False) cond
         smt_n k
+
```

Comment:
Heathen! Extra confusing newline!

Reply:
ah but i'm always so confused by how congealed the code is!! I can't tell quickly where ideas separate